src/views/v2/day/event/date.php

```diff
@@ -9,25 +9,30 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.4
+ * @version TBD
  *
  */
+use Tribe__Date_Utils as Dates;
 
 $event       = $this->get( 'event' );
 $event_id    = $event->ID;
 $is_featured = tribe( 'tec.featured_events' )->is_featured( $event_id );
+$event_date_attr = $event->dates->start->format( Dates::DBDATEFORMAT );
 
 ?>
 <div class="tribe-events-calendar-day__event-datetime-wrapper">
-	<time class="tribe-events-calendar-day__event-datetime tribe-common-b2" datetime="1970-01-01T00:00:00+00:00">
-		<?php echo tribe_events_event_schedule_details( $event ); ?>
-	</time>
 	<?php if ( $is_featured ) : ?>
 		<em
 			class="tribe-events-calendar-day__event-datetime-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
 			aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
 			title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
 		>
 		</em>
+		<span class="tribe-events-calendar-day__event-datetime-featured-text tribe-common-b2">
+			<?php esc_html_e( 'Featured', 'the-events-calendar' ); ?>
+		</span>
 	<?php endif; ?>
+	<time class="tribe-events-calendar-day__event-datetime tribe-common-b2" datetime="<?php echo esc_attr( $event_date_attr ); ?>">
+		<?php echo tribe_events_event_schedule_details( $event ); ?>
```

Comment:
Use the `schedule_details` property here.
```suggestion
		<?php echo esc_html( $event->schedule_details->value() ); ?>
```